client/my-sites/plugins/main.jsx

```diff
@@ -386,6 +387,7 @@ const PluginsMain = React.createClass( {
 		return (
 			<Main className={ containerClass }>
 				{ this.renderDocumentHead() }
+				<QuerySites allSites />
```

Comment:
We really should be avoiding `<QuerySites allSites />` calls until we slay SitesList. What behavior were you seeing that required us to add this? Seems like prod should be broken too otherwise.